pages/user guides/discover_nmap.md

```diff
@@ -8,58 +8,50 @@ permalink: discover-nmap.html
 ---
 
 
+As part of your cloud migration journey it is important to have all the tools at your disposal. In addition to [the other discovery methods](https://guides.tidalmg.com), you can use [Nmap](https://nmap.org/) to facilitate a part of the discovery process.
+The way it works is very simple and with the full power of Nmap available to you, discovery of even those hard to reach places is possible.
+
+Nmap has developed some powerful capabilities and good [documentation](https://nmap.org/book/host-discovery-find-ips.html) that will allow you to identify all the hosts you have running on your network, the ports and services that they have open and running.
 
-As part of your cloud migration journey it is important to have all the tools at your disposal. In addition to [the other discovery methods](https://guides.tidalmg.com), you can use [Nmap](https://nmap.org/) to facilitate a part of the discovery process. 
-The way it works is very simple and with the full power of Nmap available to you, discovery of even those hard to reach places is possible. 
- 
-Nmap has developed some powerful capabilities and good [documentation](https://nmap.org/book/host-discovery-find-ips.html) that will allow you to identify all the hosts you have running on your network, the ports and services that they have open and running. 
- 
 Nmap is capable of producing its output in an XML file, allowing you to inspect the raw scan output before sending it to the Tidal Migrations API with [Tidal Tools](https://get.tidal.sh). Once uploaded to the Tidal Migrations Platform, you will be able to visualize your network devices, track your complete server inventory and build on this data with other discovery methods, allowing you to make informed decisions on your cloud migration path.
```

Comment:
Long sentence. Let' rewrite it something like the following:

Nmap is capable of producing its output in an XML file. It allows you to inspect the raw scan output before sending it to the Tidal Migrations API with Tidal Tools.  Once uploaded to the Tidal Migrations Platform, you will be able to visualize your network devices, track your complete server inventory and build on this data with other discovery methods. That's how you can make informed decisions on your cloud migration path.

Reply:
nice, I like how that last sentence ("That's how you can...").